### DISCUSSION TITLE
【任务】2023年7-8月社区激励任务｜50,000 $PEOPLE

### DISCUSSION THREAD

**Discussion by AmazingAng** [AmazingAng/WTF-Solidity#530] [任务 | Bounty]
WTF Academy 社区激励任务，包括但不限于审核，改善WTF极简教程，也包括日常的PR/issue。这部分奖励优先分配给月度社区激励任务，剩余分给当月贡献者。

## 黑客松: web3 quiz应用

目标是完成一个web3 quiz应用，可以创建测试并收集答题，且不会公开答案。这个项目会参与8月 ETH Global 举办的 SuperHack 线上黑客松，然后用于 wtf.academy 教育平台。

要求:
- 合约: 熟悉solidity，熟悉ethereum attestation service是加分项
- 后端: 熟悉 go 和 solidity
- 前端: 熟悉 react 和 ethersjs
- 设计: 熟悉网页设计/figma

**Comment by hattiepwvi** [comment]
我想报名🐶

**Reply by AmazingAng** [reply]
来啊

**Comment by alex-x-crypto** [comment]
想报名，熟悉后端、Go、Solidity

**Reply by AmazingAng** [reply]
好的 有空聊一下

**Reply by AmazingAng** [reply]
你的wx多少

**Reply by alex-x-crypto** [reply]
I---alex---I

**Comment by skyf0cker** [comment]
想报名做后端，熟悉 Go 了解 Solidity 可以简单写写，也可以简单使用 hardhat, ethers.js 等 module 和 remix 等工具。
可以 tg 联系我：https://t.me/vophan

**Comment by hash-panda** [comment]
这个项目的进展如何了？
